src/main/scala/com/amazon/deequ/checks/ColumnCondition.scala

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. A copy of the License
+ * is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.checks
+
+object ColumnCondition {
+  def isNonNull(col: String): String = {
+    s"$col IS NOT NULL"
+  }
+
+  def isEachNonNull(cols: Seq[String]): String = {
```

Comment:
Please dont build the predicate via string concatenation, which is error prone. Please use Spark's functions from org.apache.spark.sql.function for constructing this.

Reply:
all right, will do

Reply:
done